DOCS/man/mpv.rst

```diff
@@ -270,6 +270,34 @@ Alt+2 (and Command+2 on macOS)
 Command + f (macOS only)
     Toggle fullscreen (see also ``--fs``).
 
+(The following keybindings open a selector in the console that lets you choose
+from a list of items by typing part of the desired item and/or by navigating
+them with keybindings: ``Down`` and ``Ctrl+n`` go down, ``Up`` and ``Ctrl+p`` go
+up, ``Page down`` and ``Ctrl+f`` scroll down one page, and ``Page up`` and
+``Ctrl+b`` scroll up one page.)
+
+g-p
+    Select a playlist entry.
+
+g-t
+    Select a track.
+
+g-j
+    Select a secondary subtitle.
+
+g-c
+    Select a chapter.
+
+g-s
+    Select a subtitle line to seek to. This requires ``ffmpeg`` in PATH, or in
```

Comment:
I'm not sure about this option. All those external subprocesses to just get subtitle list looks like a hack to me.

Reply:
I think we should implement it as a property later to remove the ffmpeg dependency. It's been discussed in https://github.com/mpv-player/mpv/issues/12810

Reply:
I won't reject this change, because I recognize value of searching subtitle, but I feel unease about suprocess running like that. I know it is common in scripts, but first party integration should be smarter.

Reply:
Agreed but this is the best we can do until someone is willing to implement a property.

Reply:
We do cache subtitle packets now so the information is there. Of course, it can get dropped because of a sub track switch, etc.